internal/serve/httphandler/receiver_wallets_handler.go

```diff
@@ -0,0 +1,36 @@
+package httphandler
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/stellar/go/support/render/httpjson"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/data"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/db"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/serve/httperror"
+)
+
+type ReceiverWalletsHandler struct {
+	Models           *data.Models
+	DBConnectionPool db.DBConnectionPool
+}
```

Comment:
There's no need to receive the `DBConnectionPool` here, you can get it from the `data.Models`.

```suggestion
type ReceiverWalletsHandler struct {
	Models           *data.Models
}
```